### PR TITLE
[plugin.video.redbull.tv@matrix] 3.2.0+matrix.1

### PR DESCRIPTION
**Pull request by piejanssens**
### Add-on details:

- **General**
  - Add-on name: Red Bull TV
  - Add-on ID: plugin.video.redbull.tv
  - Version number: 3.2.0+matrix.1
  - Kodi/repository version: matrix

- **Code location**
  - URL: https://github.com/add-ons/plugin.video.redbull.tv
  
Red Bull TV gives you front-row access to live events, the best inaction sports, new music and entertainment, and thrilling videos from world adventurers.

### Description of changes:

v3.2.0
- Support VOD playback from EPG via context menu
- Add YouTube integration (requires YouTube add-on)
- Improve channel listing
- New add-on icon

v3.1.0
- IPTV Manager support

v3.0.0
- Harder, Better, Faster, Stronger
    

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: plugin.video.redbull.tv/addon.xml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  <requires>
+    <import addon="script.module.dateutil" version="2.8.0"/>
+    <import addon="script.module.routing" version="0.2.0"/>
+    <import addon="xbmc.python" version="3.0.0"/>
+  </requires>
+  <extension point="xbmc.python.pluginsource" library="resources/lib/addon_entry.py">
+    <provides>video</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en">Watch Red Bull TV, live and on demand.</summary>
```
Comment (enen92) [inline plugin.video.redbull.tv/addon.xml]: en_GB here and the 2 lines below

File: plugin.video.redbull.tv/addon.xml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    <description lang="en">Red Bull TV gives you front-row access to live events, the best inaction sports, new music and entertainment, and thrilling videos from world adventurers.</description>
+    <disclaimer lang="en">This add-on is not endorsed by Red Bull GmbH and is provided 'as is' without any warranty of any kind.
+    </disclaimer>
```
Comment (enen92) [inline plugin.video.redbull.tv/addon.xml]: License field missing, please add one with a spdx identifier

File: plugin.video.redbull.tv/resources/lib/addon.py [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+        TitleItem(
+            title=localize(30010),  # A-Z
+            path=plugin.url_for(iptv_play),
+            art_dict=dict(
+                icon='DefaultMovieTitle.png',
+                fanart=addon_fanart(),
+                poster=addon_icon()
+            ),
+            info_dict=dict(
+                plot='Best of Red Bull TV',
```
Comment (enen92) [inline plugin.video.redbull.tv/resources/lib/addon.py]: Please localize

File: plugin.video.redbull.tv/resources/lib/addon.py [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+            list_items.append(generate_list_item(item, PRODUCT))
+
+    if not list_items:
+        ok_dialog(localize(30223), localize(30224), localize(30225))
+        return
+
+    for list_item in list_items:
+        addDirectoryItem(handle=plugin.handle, url=list_item.getPath(), listitem=list_item, isFolder=('/play/' not in list_item.getPath()))
+
+    executebuiltin('Container.SetViewMode({mode})'.format(mode=55))  # Wide List
```
Comment (enen92) [inline plugin.video.redbull.tv/resources/lib/addon.py]: Not allowed in the repo, please drop this call

**Review by enen92 — CHANGES_REQUESTED** [review]

**Comment by piejanssens** [comment]
Hi @enen92,

Thanks for your review. 
I belief I have addressed your remarks. for both the Leia and Matrix PR.

P

**Review by enen92 — APPROVED** [review]

**Pull request merged**